main/autogen/subclasses.cc

```diff
@@ -112,19 +112,17 @@ vector<string> Subclasses::serializeSubclassMap(const Subclasses::Map &descendan
         auto type = children.classKind == ClassKind::Class ? "class" : "module";
         descendantsMapSerialized.emplace_back(fmt::format("{} {}", type, parentName));
 
-        vector<string> serializedChildren;
+        auto subclassesStart = descendantsMapSerialized.size();
```

Comment:
Is it worth adding `descendantsMapSerialized.resize(descendantsMapSerialized.size() + children.entries.size())` here as well?

Reply:
My sense is no: the `reserve`-up-front strategy makes sure that you don't do a bunch of copying over the course of the entire loop.  We can't pre-reserve all the space here, so one way or the other, we have to eat the cost of copying in the loop body itself.  The current way accepts that cost and is shorter.  (Presumably we're not going to resize multiple times during this inner loop...although I guess we might in the early stages of finding subclasses.)